app/Css/About.hs

```diff
@@ -8,8 +14,7 @@ import Prelude hiding ((**))
 import Data.Monoid
 import Css.Constants
 
-{- aboutStyles
- - Generates CSS for the about page. -}
+-- |Returns the CSS for the about page.
```

Comment:
Don't say `returns` to describe nullary (0-parameter) functions. Just say that this value *is* the CSS for the about page.